components/Footer.vue

```diff
@@ -21,6 +21,9 @@
 	</b-row>
 </template>
 <script>
+import ContactInfo from "~/components/ContactInfo"
+import SocialMedia from "~/components/SocialMedia"
+import Funders from "~/components/Funder"
```

Comment:
I think you need to change this to "~components/Funders".

Reply:
`~/components/Funder` is the correct way to reference the address of a component.